apps/web/modules/survey/components/question-form-input/components/recall-wrapper.test.tsx

```
@@ -29,53 +41,37 @@ vi.mock("@/lib/utils/recall", async () => {
 vi.mock("@/modules/survey/components/question-form-input/components/fallback-input", () => ({
-  FallbackInput: vi.fn().mockImplementation(({ addFallback }) => (
-    <div data-testid="fallback-input">
-      <button data-testid="add-fallback-btn" onClick={addFallback}>
-        Add Fallback
-      </button>
-    </div>
-  )),
+  FallbackInput: vi.fn().mockImplementation(({ addFallback, open }) =>
+    open ? (
+      <div data-testid="fallback-input">
+        <button data-testid="add-fallback-btn" onClick={addFallback}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)